package.json

```diff
@@ -38,6 +38,8 @@
     "chart.js": "^4.3.0",
     "dayjs": "^1.11.8",
     "framer-motion": "^10.12.8",
+    "graphql": "^16.7.1",
+    "graphql-request": "^5.0.0",
```

Comment:
why not using the latest major version? https://www.npmjs.com/package/graphql-request?activeTab=versions

Reply:
react-query recommends using the 5.x branch for graphq-request:
- https://tanstack.com/query/v4/docs/react/graphql
- https://the-guild.dev/graphql/codegen/docs/guides/react-vue
that said, i'm not sure why and it could be good to research why that is the case.

Reply:
ok good to know

Reply:
Opened #123 for tracking.